pycbc/inject/inject.py

```diff
@@ -541,6 +564,14 @@ def apply(self, strain, detector_name, f_lower=None, distance_scale=1,
         if self.table[0]['approximant'] in fd_det:
             t0 = float(strain.start_time)
             t1 = float(strain.end_time)
+        elif detector_name in get_available_space_detectors():
+            au_travel_time = lal.AU_SI / lal.C_SI
+            try:
+                arm_travel_time = _space_detectors[detector_name]['armlength'] / lal.C_SI
```

Comment:
This is where you'll  need to think about what the time coordinate means in the injection file. Is it in the SSB time or the detector frame? If the SSB frame, this isn't the right time to consider. If the detector frame, it is close, but do we want to rely on the detector geometry having a single fixed arm length argument (this is a candidate for the detector class providing this information if it is really needed).

Reply:
I guess the choice here is safe though, assuming the orbit is at an AU (probably ok?).